test/TestUserAcceptance.hs

```diff
@@ -788,30 +836,19 @@ testUserCanAbortMailComposition =
   withTmuxSession "user can abort composing mail" $
         \step -> do
             startApplication
-            liftIO $ step "start composition"
-            sendKeys "m" (Literal "From")
-
-            liftIO $ step "enter from email"
-            sendKeys "Enter" (Literal "To")
-
-            liftIO $ step "enter to: email"
-            sendKeys "user@to.test\r" (Literal "Subject")
-
-            liftIO $ step "enter subject"
-            sendKeys "test subject\r" (Literal "~")
-
-            liftIO $ step "enter mail body"
-            sendKeys "iThis is a test body" (Literal "body")
+            composeNewMail step
 
-            liftIO $ step "exit insert mode in vim"
-            sendKeys "Escape" (Literal "body")
+            liftIO $ step "abort mail"
+            sendKeys "q" (Literal "Keep draft?")
 
-            liftIO $ step "exit vim"
-            sendKeys ": x\r" (Regex $ "From: "
-                             <> "\"Joe Bloggs\" <joe@foo.test>")
+            liftIO $ step "choose discard"
+            --TODO: buildAnsiRegex will cause the generated Regex not
```

Comment:
This one was weird: I couldn't use the regex to match the raw output and there was nothing obviously wrong with it.